packages/account/tsconfig.json

```
@@ -17,5 +17,5 @@
             "@deriv/*": ["../*/src"]
         }
     },
-    "include": ["src", "globals.d.ts", "../../global.d.ts", "../../utils.d.ts"]
+    "include": ["src", "globals.d.ts"]
```

```suggestion
    "include": ["src"]
```
```suggestion
    "include": ["src"]
```
@hamid-deriv You can remove the file too as it's the same as `svg.d.ts` which you added.